test/integration/memcache/inspec.yml

```diff
@@ -27,4 +27,6 @@ attributes:
   - name: output_nodes
     required: true
     type: hash
-
+  - name: output_discovery
```

Comment:
nit: we dont need to plumb this into inpec if we are not using it in tests

Reply:
my bad, i was just following what a previous pr had done

Reply:
Resolving since I don't think this is a problem to include.